crs_motion_planning/include/crs_motion_planning/path_planning_utils.h

```diff
@@ -47,6 +47,8 @@ struct descartesConfig
   bool allow_collisions = false;
 
   Eigen::Isometry3d tool_offset = Eigen::Isometry3d::Identity();
+
+  std::vector<double> joint_min_vals;
 };
 
 struct trajoptSurfaceConfig
```

Comment:
Just realized that this struct does not adhere to naming conventions, first character should be in capitalized